src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
```

Comment:
Consider checking if the injector instance is not null before using it. This can prevent potential NullPointerExceptions if the Injector.getInjector() method ever returns null.